blog/en/blog/2023/06/08/weekly-report.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Biweekly Report (May 23 - June 8)"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/06/08/njsXFE15_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5%E5%A4%B4%E5%9B%BE-%E8%8B%B1%E6%96%87.png
+---
+
+> From 5.23 to 6.05, 20 contributors submitted 37 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://security.feishu.cn/link/safety?target=https%3A%2F%2Fstatic.apiseven.com%2Fuploads%2F2023%2F06%2F08%2FG1yIQjyj_%25E6%2589%2580%25E6%259C%2589%25E8%25B4%25A1%25E7%258C%25AE%25E8%2580%2585%25E6%25B5%25B7%25E6%258A%25A5.png&scene=ccm&logParams=%7B%22location%22%3A%22ccm_default%22%7D&lang=zh-CN)
```

Comment:
```suggestion
![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/06/09/XavQu9xL_contributor%20list0609.png)
```

Reply:
Please align the names to make the picture look more neat! You can use this one this time.